{{ cookiecutter.project_name }}/tox.ini

```diff
@@ -8,6 +8,7 @@ setenv =
     SECRET_KEY='FOO'
     ALLOWED_HOSTS=localhost
     DATABASE_URL=postgres://localhost/{{ cookiecutter.project_db }}
+    SECURE_SSL_REDIRECT=False
```

Comment:
This is needed because the default for `SECURE_SSL_REDIRECT` is `not DEBUG`. I guess when hacking you have `Debug=True` and tests don't fail.